pogobuf/pogobuf.client.js

```diff
@@ -154,12 +155,7 @@ function Client(options) {
      * @return {Promise}
      */
     this.batchCall = function() {
-        if (!self.batchRequests || !self.batchRequests.length) {
-            return Promise.resolve(false);
-        }
-
-        var p = self.callRPC(self.batchRequests);
-
+        var p = self.callRPC(self.batchRequests || []);
```

Comment:
Currently you cannot send a request without api calls, however the app does it for its very first call.